src/averageAges.js

```diff
@@ -20,6 +20,16 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
+  let onlyMen = people.filter(person => person.sex === 'm');
+
+  if (arguments.length === 2) {
+    onlyMen = onlyMen.filter(man => Math.ceil(man.died / 100) === century);
+  }
```

Comment:
avoid using loop and forEach
  // replace `if ()` statement with &&, || or ?:
  // without nesting
  
  you can use ?: here